guides/source/5_0_release_notes.md

```diff
@@ -227,6 +227,10 @@ Please refer to the [Changelog][action-pack] for detailed changes.
     `RedirectBackError`.
     ([Pull Request](https://github.com/rails/rails/pull/22506))
 
+*   `ActionDispatch::IntegrationTest` and
+    `ActionController::TestCase` accept keyword arguments instead of
+positional arguments. ([Pull Request](https://github.com/rails/rails/pull/18323))
```

Comment:
Alignment is off because `positional` word is not aligned with `ActionDispatch`. See the lines above